cfgov/cal/event.py

```diff
@@ -1,10 +1,12 @@
-from cal.models import CFPBCalendar, CFPBCalendarEvent
-from datetime import date, datetime
+from cal.models import CFPBCalendarEvent
+from datetime import datetime
+import re
```

Comment:
So, flake8-import-order doesn't support Django's particular import styles. I've cleaned up all the imports though in 37df319 (for the all the apps in the project that I'm touching so far). But I'd still love a good way to check imports on an ongoing basis.